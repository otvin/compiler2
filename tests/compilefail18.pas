program compilefail18(output);
{parse failure - function missing return type}

function addone(r:integer);
begin
	addone:=r+1;
end;

begin
	writeln(addone(3));
end.
