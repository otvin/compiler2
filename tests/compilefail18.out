Function addone missing return type at file:tests/compilefail18.pas line:4     column:1  
